package.json

```diff
@@ -91,7 +92,7 @@
     "test:unit": "mocha ./test/unit/index.js",
     "test:integration:adapter": "node ./scripts/karma.js karma/integration.adapter.conf.js",
     "test:integration": "node ./scripts/karma.js karma/integration.conf.js",
-    "test:umd:install": "npm install puppeteer",
+    "test:umd:install": "npm install puppeteer@5.5.0",
```

Comment:
@manjeshbhargav @makarandp0 @PikaJoyce I added this version because puppeteer released a major version with breaking changes recently which broke our tests. I'm locking the version so it doesn't auto update. Let me know if you have any concerns.